sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -569,7 +577,18 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 					"Marking it as '%s' in the network graph.",
 					container.Namespace, container.ContainerName, conn.remote.IPAndPort.String(), entitiesName)
 			}
+
+			if !status.used {
+				if isExternal {
+					flowMetrics.ExternalFlowCounter.With(metricDirection).Inc()
+				} else {
+					flowMetrics.InternalFlowCounter.With(metricDirection).Inc()
+				}
+			}
 		} else {
+			if !status.used {
+				flowMetrics.NetworkEntityFlowCounter.With(metricDirection).Inc()
```

Comment:
I am afraid, this is insufficient - remember, we are inside of `if len(lookupResults) == 0 {`. We should _additionally_ count the flows for which we found the entities in the lookup - see the `else` branch in line 599.

Reply:
> We should additionally count the flows for which we found the entities in the lookup

Should we?  If `len(lookupResults) > 0` doesn't that mean that we know *both* entitites in this connection? I don't see how this should increment the counter on external or internal network flows as it won't produce an edge in the graph between the node being processed and the external/internal node.

Reply:
Your comment is fully correct - this will not have influence on internal and external, but it will count the other flows incorrectly (through `NetworkEntityFlowCounter`).

Reply:
ok this makes sense. Updated the code

Reply:
This looks good now, thanks!